src/components/LHNOptionsList/LHNOptionsList.tsx

```diff
@@ -111,9 +119,53 @@ function LHNOptionsList({
 
     const extraData = useMemo(() => [reportActions, reports, policy, personalDetails, data.length], [reportActions, reports, policy, personalDetails, data.length]);
 
+    const previousOptionMode = usePrevious(optionMode);
+
+    useEffect(() => {
+        if (previousOptionMode === null || previousOptionMode === optionMode || !flashListRef.current) {
+            return;
+        }
+
+        if (!flashListRef.current) {
+            return;
+        }
+
+        // If the option mode changes want to scroll to the top of the list because rendered items will have different height.
```

Comment:
why are we looking at the `optionMode` to determine if the scroll position should be adjusted?

Reply:
If we change the option mode, the elements will have different heights. Especially if the user scrolled to the bottom of the list chat, this difference will add up and the elements will move. It may be confusing for the user.

Reply:
ah I now completely understand. thanks for the explanation 🙇